### ISSUE TITLE
no longer able to open IX-Select when used in a region that gets hidden and shown again

### ISSUE THREAD

**Issue by neunteufelflorian** [siemens/ix#1134]
### Prerequisites

- [X] I have not leaked any internal company information.
- [X] I have not posted internal links or code snippets.

### What happened?

We are using an IX-Select to set a specific View Setting.
It is used inside one of our regions that the user is able to display on a button click and hide again on clicking the close-icon:

![image](https://github.com/siemens/ix/assets/119290892/fc6e323b-b34c-4948-b733-5fc7f43ee18f)

When first opening it, selecting a value from the Select works just fine, but once we close the region, open it again and try selecting something, the Dropdown is no longer expanding and we are no longer able to select a value.

This was not occurring in IX 2.0 before and happened just after we updated to 2.1 (2.1.1)

### What type of frontend frameware are you seeing the problem on?

Angular

### Which version of iX do you use?

v2.1.1

### Code to produce this issue.

```shell
<ix-select [value]="selectedItem" data-test-id="selectControl">
    <ix-select-item
      *ngFor="let item of filterTileSelectionList"
      [label]="item.label"
      [value]="item.value"
      (itemClick)="selectItemClick(item)"
      [attr.data-test-id]="item.value"
    >
    </ix-select-item>
  </ix-select>
```

**Comment by neunteufelflorian** [comment]
Short Update from my side:

Using IX 2.0 has almost the same behaviour as 2.1:

On closing a Region where the Select is used in Version 2.0, the Dropdown will no longer disappear on opening again, whilst on 2.1, as stated above, the dropdown is no longer opening 😄 

For 2.0 console errors look like this:
![image](https://github.com/siemens/ix/assets/119290892/efcb39c6-a22e-4c01-a63a-1ecfb1c45417)

2.1 doesn't provide any console errors, as I am not even able to interact in any way with the select anymore it seems 😄

**Comment by matthiashader** [comment]
Fixed in  #1192
